packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js

```diff
@@ -75,7 +79,9 @@ import {
   getEngineCategories
 } from '.';
 
-import { ADD_DETECTED_FACE } from './faceEngineOutput';
+import {
+  ADD_DETECTED_FACE
```

Comment:
this should be included with the previous import declaration from faceEngineOutput